CHANGELOG

```diff
@@ -1,3 +1,10 @@
+v0.6.0 (06/17/16)
+* Support Python 3
+* Use pytest to run tests
+* Save browser page source on failure
+* Allow configuration of the firefox profile
```

Comment:
nit: I'd reword it to something like "allow firefox profile to be set from a file on disk" or something like that, that makes it clear it's an already-saved profile, rather than one which is programmatically created.